From ae6098d54de94186d3a6a8651ff4cd85aa68f0c9 Mon Sep 17 00:00:00 2001
From: Vit Mojzis <vmojzis@redhat.com>
Date: Wed, 15 Nov 2017 15:02:07 +0100
Subject: [PATCH 1/2] framework/setroubleshoot: Do not change if_string[0] to
 lowercase

The change to lowercase was left over from the previous format where the
if_string didn't contain the leading "If " string.

Signed-off-by: Vit Mojzis <vmojzis@redhat.com>
---
 framework/src/setroubleshoot/signature.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/framework/src/setroubleshoot/signature.py b/framework/src/setroubleshoot/signature.py
index 506ea23..ec7c934 100755
--- a/framework/src/setroubleshoot/signature.py
+++ b/framework/src/setroubleshoot/signature.py
@@ -574,7 +574,7 @@ class SEFaultSignatureInfo(XmlSerialize):
                 text +=  _("*")
             text +=  _("\n")
             txt = self.substitute(p.get_if_text(self.audit_event.records, args)).decode('utf-8')
-            text +=  _("\n") + txt[0].lower() + txt[1:]
+            text +=  _("\n") + txt
             txt = self.substitute(p.get_then_text(self.audit_event.records, args)).decode('utf-8')
             text +=  _("\nThen ") + txt[0].lower() + txt[1:]
 
-- 
2.9.4

